link_shortener/core/initialise_db.py

```diff
@@ -97,14 +99,24 @@
 
 @initdb_blueprint.listener('before_server_start')
 async def initialise_db(app, loop):
-    app.engine = await create_engine(
-        host='db',
-        port=3306,
-        user='user',
-        password='password',
-        db='db',
-        loop=loop
-    )
+    if config('PRODUCTION', default=False, cast=bool):
+        app.engine = await create_engine(
+            host=config('MYSQL_HOST'),
+            port=config('MYSQL_PORT', cast=int),
+            user=config('MYSQL_USER'),
+            password=config('MYSQL_PASSWORD'),
+            db=config('MYSQL_DB'),
+            loop=loop
+        )
+    else:
+        app.engine = await create_engine(
+            host='db',
+            port=3306,
+            user='user',
+            password='password',
+            db='db',
+            loop=loop
+        )
```

Comment:
What about having just this:
```suggestion
    app.engine = await create_engine(
        host=config('MYSQL_HOST'),
        port=config('MYSQL_PORT', cast=int),
        user=config('MYSQL_USER'),
        password=config('MYSQL_PASSWORD'),
        db=config('MYSQL_DB'),
        loop=loop
    )
```
? I mean, if people want to run it against localhost, they would set the credentials in `settings.ini` accordingly. But if you prefer this way, I'm fine with that...

Reply:
It's a DEV/PROD switch for me, so I only have to change one variable in settings.ini between building the image and developing new features.